emissionsapi/db.py

```diff
@@ -194,6 +196,11 @@ def filter_query(query, polygon=None, begin=None, end=None):
         query = query.filter(geoalchemy2.func.ST_WITHIN(
             Carbonmonoxide.geom, polygon))
 
+    # Filter with point, 0.2 decimal degrees are approx. 22264 meter
+    if point is not None:
+        query = query.filter(geoalchemy2.func.ST_DWITHIN(
+            Carbonmonoxide.geom, point, 0.2))
```

Comment:
would it make sense to let users specify this value instead of hard-coding it?

Reply:
Yes, it could be reasonable in the future to let the user specify this. At least the filter function should get it as a input parameter, so that we can easier add this in the future.